### ISSUE TITLE
Narrator doesn't announce tab selection in pivot control in Edge browser

### ISSUE THREAD

**Issue by gaurav42** [OfficeDev/office-ui-fabric-react#5447]
### Bug Report 
- __OS__: Version 1803 (OS Build 17134.5)
- __Build Version__: Microsoft Edge 42.17134.1.0
- __Screen Reader__: Narrator,NVDA
 
#### Priorities and help requested:
 
Are you willing to submit a PR to fix? No
 
Requested priority: Normal
 
#### Describe the issue:
 
In pivot control, narrator doesn't announce the selected state of tab items when Enter key is pressed. It announces only when user navigates using arrow keys.

Issue can be reproed on Office fabric demo site:
https://developer.microsoft.com/en-us/fabric#/components/pivot 
 
#### Actual behavior:
 
When user navigates to tab, tab name is announced as not selected. But when user selects that tab by pressing Enter key, narrator doesn't announce that tab is selected.
 
#### Expected behavior:
 
When user selects tab by pressing Enter key, narrator should announce the selected tab.

**Comment by JasonGore** [comment]
This is also the behavior of Narrator / Edge with [ARIA tab reference examples](https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-2/tabs.html). If you feel this is required behavior, then an issue can be reported to Narrator as shown in the [wiki](https://github.com/OfficeDev/office-ui-fabric-react/wiki/Accessibility-Troubleshooting). As this is not an issue specific to Fabric, I will close this issue.
